core/services/keystore/encryption.go

```diff
@@ -0,0 +1,162 @@
+package keystore
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/keys/encryptionkey"
+)
+
+// ErrEncryptionKeyExists describes the error when the Encryption key already exists
+var ErrEncryptionKeyExists = errors.New("can only have 1 Encryption key")
+
+type Encryption interface {
```

Comment:
Encryption is a very general name.

Could you please provide something that will distinguish this interface from other key storages (CSA, Eth, OCR, OCR2, P2P, Cosmos, Solana, StarkNet, Aptos, VRF)?